src/components/learning-center/all-tools-cta.tsx

```diff
@@ -1,56 +1,34 @@
 import React from 'react'
-import { StaticQuery, graphql } from "gatsby";
 import { ContentfulContent } from '../../pages';
 import { DDOSearchBar } from '../ddo-searchbar';
 
 const widont = require('widont')
```

Comment:
Heh nice, on the tenant platform I made a funky React helper library called [`word-glue.tsx`](https://github.com/JustFixNYC/tenants2/blob/master/frontend/lib/word-glue.tsx) that is somewhat similar to this.

Reply:
haha cool! I kinda love the name "widont" though, it sounds so sassy to me